chapter/simpleTypes.tex

```diff
@@ -6,6 +6,9 @@ \section{Simply Typed Lambda Calculus}
 \newcommand\CaseTerm[3]{\texttt{case}\;#1\;\texttt{of}\;\iota_1\;x.#2\texttt{|}\iota_2\;x.#3}
 \newcommand\iotaval{\iota^{\textsc{v}}}
 
+As pure lambda calculus doesn't have any typeable terms, we need to extend it with
+some atomic values $c_b$ of type $b$, which can for example be numbers or booleans.
```

Comment:
This is not exactly true. We add some base types, in order to have non-empty grammar of types. But if we add some base types, we observe that they may have some elements (like type `Bool` has two constants associated with it).